controllers/userControllers.js

```diff
@@ -0,0 +1,44 @@
+import { Todos } from '../models/todo.js';
+
+
+const insertActionTodo = async (req, res) => {
+    const { username, group, subtask, isUrgent, isDone, hours } = req.body;
```

Comment:
Move all the logic to the services folder. Controllers should not have this much logic.